packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -388,32 +392,56 @@ export default class CodeMode extends Component<Signature> {
         try {
           if (isReady(this.openFile.current) && this.importedModule?.module) {
             let module = this.importedModule?.module;
-            let cardsOrFields = cardsOrFieldsFromModule(module);
+            let exportedCardsOrFields = cardsOrFieldsFromModule(module);
             let elements: Element[] = [];
             await this.importedModule.loaded;
             let moduleSyntax = new ModuleSyntax(this.openFile.current.content);
-            elements = moduleSyntax.elements.map(
-              (value: ElementDeclaration) => {
-                let cardOrField = cardsOrFields.find(
+            // This loop
+            // - collects super / fields from cards / fields that have been exported
+            let localCardsOrFields = collectLocalCardsOrFields(
+              moduleSyntax,
+              exportedCardsOrFields,
+            );
+
+            // This loop
+            // - adds card type (not loaded tho)
+            // - includes card or field
+            //   - an already exported card or field
+            //   - an already that was local but exported thru some relationship
+            elements = moduleSyntax.elements.map((value) => {
+              if (isPossibleCardOrFieldClass(value)) {
+                const cardOrField = exportedCardsOrFields.find(
                   (c) => c.name === value.localName,
                 );
-                if (cardOrField !== undefined) {
+                if (cardOrField) {
                   return {
                     ...value,
+                    cardOrField,
                     cardType: getCardType(
                       this,
                       () => cardOrField as typeof BaseDef,
                     ),
-                    cardOrField,
                   } as CardOrField & Partial<PossibleCardOrFieldClass>;
+                } else {
+                  if (localCardsOrFields.has(value)) {
+                    let cardOrField = localCardsOrFields.get(
+                      value,
+                    ) as typeof BaseDef;
+                    return {
+                      ...value,
+                      cardOrField,
+                      cardType: getCardType(this, () => cardOrField),
+                    } as CardOrField & Partial<PossibleCardOrFieldClass>;
+                  }
                 }
-                return value as BaseDeclaration;
-              },
-            );
+              }
+              return value as BaseDeclaration;
+            });
             state.value = elements;
           }
         } catch (error: any) {
           state.error = error;
+          console.log(error);
```

Comment:
i always use `console.error()` to log errors so they appear in red in the console